released/discovery_center/mission_4327/setup_subaccount_cf/main.tf

```diff
@@ -26,16 +27,26 @@ resource "btp_subaccount_role_collection_assignment" "subaccount-admins" {
 ######################################################################
 # Creation of Cloud Foundry environment
 ######################################################################
-module "cloudfoundry_environment" {
-  source                    = "../../../modules/environment/cloudfoundry/envinstance_cf"
-  subaccount_id             = btp_subaccount.project.id
-  instance_name             = local.project_subaccount_cf_org
-  plan_name                 = "standard"
-  cf_org_name               = local.project_subaccount_cf_org
-  cf_org_auditors           = var.cf_org_auditors
-  cf_org_managers           = var.cf_org_managers
-  cf_org_billing_managers   = var.cf_org_billing_managers
+resource "btp_subaccount_environment_instance" "cloudfoundry" {
```

Comment:
This is a functional change as the CF org users are no longer assigned. Is this in sync with the discovery center mission?

Reply:
Yes, This is in sync with with the mission.

Please note that our Terraform configuration for setting up the BTP account in this mission does not involve creating a Cloud Foundry space. Therefore, we've thoughtfully decided to streamline our approach by removing the CF community provider from our list. The CF runtime enablement will be managed with the Terraform BTP provider.